Browser/keywords/interaction.py

```diff
@@ -735,6 +736,7 @@ def select_options_by(
             self.deselect_options(selector)
             return []
 
+        logger.info(f"Selects the option(s) {', '.join(values)} by attribute {attribute} from element {selector}.")
```

Comment:
I think this breaks the keyword, because values can contains something else than strings.

Reply:
Better?

Reply:
Let’s see what test do say